src/internal_events/http.rs

```diff
@@ -129,3 +129,39 @@ impl<'a> InternalEvent for HttpDecompressError<'a> {
         counter!("parse_errors_total", 1);
     }
 }
+
+pub struct HttpInvalidRouteError {}
+
+impl InternalEvent for HttpInvalidRouteError {
+    fn emit(self) {
+        error!(
+            message = "Invalid route specified.",
```

Comment:
```suggestion
            message = "Request received on unknown route.",
```
I don't like the term "specified" here, sounds like a config error.

Reply:
(Feel free to tweak that, just first thought)